site/index.html

```diff
@@ -1,10 +1,55 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     <head>
-        <title></title>Hello Coursera!</title>
+        <title>Hello Coursera!</title>
     </head>
 <body>
-<h1>Hello Coursera! It's a great course!</h1>
+<h1> Hello Coursera! </h1>
 
+<div>
+<input type="text" placeholder="type here">
+<button type="submit">SUBMIT</button>
+</div>
+<div>
+<p>list</p>
+  <ol>
+      <li>first item</li>
+      <li>second item</li>
+  </ol>  
+list  
+<ul>
+      <li>one</li>
+      <li>two</li>
+  </ul>
+</div>
+<p>
+    &quote;dijasd;fja;dsklfj fad;skfj;adslkfjadskfj dkfj;adkljf;asdkfj ad hello my name&nbsp; is&nbsp; tess&quot; 
```

Comment:
Did you mean to sneeze while typing this?

Reply:
yes